analyzers/src/SonarAnalyzer.Common/Rules/Utilities/UtilityAnalyzerBase.cs

```diff
@@ -62,7 +62,7 @@ protected void ReadParameters(SonarCompilationStartAnalysisContext context)
             if (settings.Any() && !string.IsNullOrEmpty(outPath))
             {
                 var language = context.Compilation.Language;
-                IgnoreHeaderComments = PropertiesHelper.ReadIgnoreHeaderCommentsProperty(settings, language);
+                IgnoreHeaderComments = context.SonarLintFile().IgnoreHeaderComments;
```

Comment:
Now the file is parsed twice because the old way is still used for the Autogenerated property. I will make sure only 1 reading is done, in the next PR that is about to change the Autogenerated  reading logic.